docs/book/src/clusterctl/configuration.md

```diff
@@ -101,7 +101,7 @@ You may want to migrate to a user-managed cert-manager further down the line, af
 
 `clusterctl` looks for the label `clusterctl.cluster.x-k8s.io/core=cert-manager` on all api resources in the `cert-manager` namespace. If it finds the label, `clusterctl` will manage the cert-manager deployment. You can list all the resources with that label by running:
 ```bash
-kubectl get all -A --selector=clusterctl.cluster.x-k8s.io/core=cert-manager
+kubectl get mutatingwebhookconfiguration,validatingwebhookconfiguration,clusterrole,clusterrolebinding,crds,all -A --selector=clusterctl.cluster.x-k8s.io/core=cert-manager
```

Comment:
Very good catch, thx!

Wondering if we should go all in and use this:
```suggestion
kubectl api-resources --verbs=list -o name | xargs -n 1 kubectl get --show-kind --ignore-not-found -A --selector=clusterctl.cluster.x-k8s.io/core=cert-manager
```

Reply:
I had a look at that command, but it's very, **very** slow,  especially if you have a large amount of CRDs on the cluster. So I opted for the `kubectl get` method instead. Happy to change it if you think this is a better solution though

Reply:
It's pretty slow for me  - more than 1-2 seconds for sure with a couple of clusters deployed. But this is a complex operation and this command is the only assured way to get all of the labeled objects - I think it's probably the correct approach.

Reply:
I think for this once-in-a-management-cluster-lifetime command I would be totally fine with 1-2 minutes or maybe even more.

It's probably better to guarantee that all objects are captured vs. missing some by optimizing the runtime of this command

Reply:
The other idea (probably not worth it for this use case) would be to expose a clusterctl command to `unamanage` cert manager, using its labelling logic.

Reply:
We were looking about 3m on our prod management cluster, but you're also right, it's a one-time command so I have no issue with it being slow initially. I'll add a small warning on the docs mentioning the command may take a while to run.

I also like the idea of having an `unmanage` command to automate this process